router/io-spec.gobra

```diff
@@ -543,3 +543,48 @@ pure func uInfo(hopfields seq[io.IO_HF], currHFIdx int, consDir bool) set[io.IO_
 						hvfSet(hopfields[currHFIdx]) :
 						hvfSet(hopfields[currHFIdx-1])))
 }
+
+ghost
+decreases
+pure func ifsToIO_ifs(ifs uint16) option[io.IO_ifs]{
+    return ifs == 0 ? none[io.IO_ifs] : some(io.IO_ifs(ifs))
+}
```

Comment:
I'm starting to think whether this mapping from internal interface to `none` is correct and consistent with the rest of the original IO spec (that we got in the pdf). Can you confirm this please?